File: hobby-tracker-web-api/hobby-tracker-web-api/Database/Entities/Day.cs
namespace hobby_tracker_web_api.Database.Entities;

public class Day
{
  public int Id { get; set; }
  public bool IsCompleted { get; set; }
  public DateTime Date { get; set; }
  public Habit Habit { get; set; }
  public int HabitId { get; set; }
}

File: src/2.entities/Day.ts
class Day {
 constructor(id: number, isCompleted: boolean, date: Date, habitId: number) {
  this.id = id;
  this.isCompleted = isCompleted;
  this.date = date;
  this.habitId = habitId;
 }

 id: number
 isCompleted: boolean
 date: Date
 habitId: number
}
export default Day;